app/components/SuggestionsPanel/index.jsx

```diff
@@ -0,0 +1,73 @@
+import cx from 'classnames'
+import React from 'react'
+import { IntlMixin } from 'react-intl'
+import SuggestionsHeader from '../SuggestionsHeader'
+import SuggestionsBody from '../SuggestionsBody'
+import { assign, pick } from 'lodash'
+import { connect } from 'react-redux'
+
+const DO_NOT_RENDER = null
+
+/**
+ * Panel to search for and display suggestions.
+ */
+let SuggestionsPanel = React.createClass({
+  mixins: [IntlMixin],
+
+  propTypes: {
+    // likely want to move this switching to a higher level
+    showPanel: React.PropTypes.bool.isRequired,
+    searchType: React.PropTypes.oneOf(['phrase', 'text']).isRequired
+  },
+
+  render: function () {
+    if (!this.props.showPanel) {
+      return DO_NOT_RENDER
+    }
+
+    const className = cx('Editor-suggestions Editor-panel u-bgHigh', {
+      'is-search-active': this.props.searchType === 'text'
+    })
+
+    const headerProps = pick(this.props, ['showDiff', 'onDiffChange',
```

Comment:
Since these props are just getting passed down and not used here, shouldn't we just access them directly from `SuggestionsHeader` from the redux state?